src/main/setupInvestHandlers.js

```diff
@@ -112,50 +102,66 @@ export function setupInvestRunHandlers(investExe) {
     ];
     logger.debug(`set to run ${cmdArgs}`);
     let investRun;
+    let investLogfile;
+    const env = {
+      PATH: path.dirname(investExe),
+      PYTHONUNBUFFERED: 'TRUE',
```

Comment:
I actually removed this env var since discovering that `logging` flushes with each emit. Since our goal is to get each logger message one at a time over stdout, I think that's all we need! Do you concur? https://github.com/python/cpython/blob/main/Lib/logging/__init__.py#L1092

Reply:
Yep, I concur!  That should do the trick for us here.